tests/__init__.py

```diff
@@ -74,6 +74,7 @@ def mkosi(
             check=check,
             stdin=stdin,
             stdout=sys.stdout,
+            stderr=sys.stderr,
```

Comment:
This is the default, no need to specify it explicitly

Reply:
Ack, dropping that commit then!